zabbix-agent2/README.md

```diff
@@ -1,4 +1,4 @@
-## Zabbix Agent 2
+## Zabbix Agent 2 (7.4.6)
```

Comment:
I feel like this will be out-of-date very quickly. Its yet another thing that needs to be updated with every release, and I dont see what this brings.

Reply:
> I feel like this will be out-of-date very quickly. Its yet another thing that needs to be updated with every release, and I dont see what this brings.

There isnt any info available on what version of Zabbix-Agent its running, except for the log file. So i thought it was an easy way to show the intended version.